.github/workflows/ci-build.yml

```diff
@@ -31,8 +31,6 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [
-          "3.9",
-          "3.10",
```

Comment:
It's great that 3.9 is out, but 3.10? I see it documented in the PR description, but we still need to support 3.10 in ee and geemap until 2026-10.

https://devguide.python.org/versions/

Reply:
Added 3.10 back in.
Done.